lib/lambda_ethereum_consensus/state_transition/epoch_processing.ex

```diff
@@ -0,0 +1,21 @@
+defmodule LambdaEthereumConsensus.StateTransition.EpochProcessing do
+  @moduledoc """
+  This module contains utility functions for handling epoch processing
+  """
+
+  alias ChainSpec
+  alias LambdaEthereumConsensus.StateTransition.Accessors
+  alias SszTypes.BeaconState
+
+  @spec process_eth1_data_reset(BeaconState.t()) :: BeaconState.t()
```

Comment:
Sorry, this should still return `{:ok, BeaconState.t()}` just to be consistent across the different processing functions

Reply:
Okay. Done